### PR TITLE
Update Changelog for the new release.

### PR DESCRIPTION
**Pull request by andrykonchin**
@richardhsu Please correct me

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
+* Allow consistent reading for Scan request (@andrykonchin)
+* Use Query instead of Scan if there are no conditions for sort (range) key in where clause (@andrykonchin)
+* Support condition operators for non-key fields for Query request (@andrykonchin)
+* Support condition operators for Scan request (@andrykonchin)
+* Support additional operators `in`, `contains`, `not_contains` (@andrykonchin)
+* Support type casting in `where` clause (@andrykonchin)
+* Rename `Chain#eval_limit` to `#record_limit`  (@richardhsu)
+* Add `Chain#scan_limit`  (@richardhsu)
+* Support batch loading for Query requests (@richardhsu)
+* Support querying Global/Local Secondary Indices  (@richardhsu)
```
Comment (richardhsu) [inline CHANGELOG.md]: * Support querying Global/Local Secondary Indices in `where` clause (@richardhsu)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,36 @@
...
+* Only query on GSI if projects all (@richardhsu)
```
Comment (richardhsu) [inline CHANGELOG.md]: * Only query on GSI if projects all attributes in `where` clause (@richardhsu)

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,36 @@
...
+
+Fixes
+
+* Fix incorrect applying of default field value (#36 and #117, @andrykonchin)
+* Fix sync table creation/deletion (#160, @mirokuxy)
+* Allow to override document timestamps (@andrykonchin)
+* Fix storing empty array as nil (#8, @andrykonchin)
+* Fix `limit` handling for Query requests (@richardhsu)
+* Fix `limit` handling for Scan requests (@richardhsu)
```
Comment (richardhsu) [inline CHANGELOG.md]: Both of the `limit` fixes for issue https://github.com/Dynamoid/Dynamoid/issues/85

**Comment by andrykonchin** [comment]
Done

**Pull request merged**